api/dto/dto.go

```diff
@@ -263,14 +263,15 @@ type TimeEntriesList struct {
 
 // TimeEntryImpl DTO
 type TimeEntryImpl struct {
-	Billable     bool         `json:"billable"`
-	Description  string       `json:"description"`
-	ID           string       `json:"id"`
-	IsLocked     bool         `json:"isLocked"`
-	ProjectID    string       `json:"projectId"`
-	TagIDs       []string     `json:"tagIds"`
-	TaskID       string       `json:"taskId"`
-	TimeInterval TimeInterval `json:"timeInterval"`
-	UserID       string       `json:"userId"`
-	WorkspaceID  string       `json:"workspaceId"`
+	Billable         bool         `json:"billable"`
+	Description      string       `json:"description"`
+	ID               string       `json:"id"`
+	IsLocked         bool         `json:"isLocked"`
+	CurrentlyRunning bool         `json:"currentlyRunning"`
```

Comment:
adding this field does not break the `log in-progress` right? i dont remember how go deals with missing fields on unmarshal.

Reply:
`log in-progress` still works correctly as far as I can tell (nothing when no running entry, latest entry when there is one)